scripts/deps-ppa/static_z3.sh

```diff
@@ -179,7 +198,7 @@ This program is free software: you can redistribute it and/or modify
  Public License version 3 can be found in "/usr/share/common-licenses/GPL-3".
 EOF
 cat <<EOF > debian/changelog
-libz3-static-dev (0.0.1-1ubuntu0) saucy; urgency=low
+z3-static (0.0.1-1ubuntu0) saucy; urgency=low
 
   * Initial release.
 
```

Comment:
I think that this package split warrants an entry in the changelog. Or, if this is now treated as a new package, the date below should be updated (ot now says `Mon, 03 Jun 2019`).

Reply:
We're not even maintaining a proper changelog for the actual solidity releases, so I don't think there is much need to do it here, at least for now (these PPA dependency builds are not meant to be used anywhere other than in the release build process anyways). Although we could independently consider doing this more properly both for the actual releases and here.